README.md

```diff
@@ -1,197 +1,189 @@
 
-4. Stop the Services:
+If the command is not found, or the version is < `1.27.4`, run the following to install the latest to `/usr/local/bin/docker-compose`:
 
 ```bash
-docker-compose down
+VERSION=$(curl --silent https://api.github.com/repos/docker/compose/releases/latest | jq .name -r)
+DESTINATION=/usr/local/bin/docker-compose
+sudo curl -L https://github.com/docker/compose/releases/download/${VERSION}/docker-compose-$(uname -s)-$(uname -m) -o $DESTINATION
+sudo chmod 755 $DESTINATION
 ```
 
-## Bootstrap Container
-
-The first container to start will always be the `mocknet_bootstrap` container. The sole purpose of this container is to run a [script](https://github.com/blockstack/stacks-local-dev/blob/mocknet/setup.sh) to replace the variables in the `.template` files with the values from `.env`
-
-## API Container
-
-The API Container will run a [script](https://github.com/blockstack/stacks-local-dev/blob/mocknet/setup-bns.sh) before starting it's server. The sole purpose of this is to download (or verify the files exist) V1 BNS data. Once the download/extraction/verification has finished, the `stacks-blockchain-api` server will start up
-
-## Env Vars
+### Env Vars
+All variables used in the [`sample.env`](https://github.com/blockstack/stacks-local-dev/blob/private-testnet/sample.env) file can be modified, but generally most of them should be left as-is.
 
-All variables used in the [`sample.env`](https://github.com/blockstack/stacks-local-dev/blob/mocknet/sample.env) file can be modified, but generally most of them should be left as-is.
+### Local Data Dirs
+Directories will be created on first start that will store persistent data under `./persistent-data/<network>`
 
-## Local Data Dirs
 
-3 Directories will be created on first start that will store persistent data. Deleting this data will result in a full resync of the blockchain, and in the case of `bns-data`, it will have to download and extract the V1 BNS data again. 
-
-### Locally opened ports
-
-In this section of the [`sample.env`](https://github.com/blockstack/stacks-local-dev/blob/mocknet/sample.env) file, the values can be modified to change the ports opened locally by docker.
+## Quickstart
 
-Currently, default port values are used - but if you have a running service on any of the defined ports, they can be adjusted to any locally available port.
+`<network>` can be 1 of:
+  - mocknet
+  - testnet
+  - mainnet
+  - private-testnet
+  
 
-ex:
+1. Clone the repo locally:
 
 ```bash
-API_STACKS_BLOCKCHAIN_API_PORT_LOCAL=3999
+$ git clone -b private-testnet --depth 1 https://github.com/blockstack/stacks-local-dev ./stacks-local-dev && cd ./stacks-local-dev
```

Comment:
Do we still need custom branches?

Reply:
Good catch!
Not particularly, but in this case since we're not on the main branch yet it was necessary to add to the Readme. 
I'll remove it since it shouldn't be specific to any branch once we merge to main.